REST Bindings/openapi.yaml

```diff
@@ -4333,6 +4644,802 @@ components:
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
+
+    MATCH_epcClass:
+      name: MATCH_epcClass
+      description: "If this parameter is specified, the result will only include events that (a) have a `quantityList` or a `childQuantityList` field (that is, `ObjectEvent`, `AggregationEvent`, `TransactionEvent`, `AssociationEvent` or extension event types that extend one of those event types); and where (b) one of the EPC classes listed in the `quantityList` or `childQuantityList` field (depending on event type) matches one of the EPC patterns or URIs specified in this parameter. The result will also include QuantityEvents whose `epcClass` field matches one of the URIs specified in this parameter. The meaning of 'matches' is as specified in section 8.2.7.1.1."
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
+
+    MATCH_inputEPCClass:
+      name: MATCH_inputEPCClass
+      description: "If this parameter is specified, the result will only include events that (a) have an `inputQuantityList` field (that is, `TransformationEvent` or extension event types that extend it); and where (b) one of the EPC classes listed in the `inputQuantityList` field (depending on event type) matches one of the EPC patterns or URIs specified in this parameter. The meaning of 'matches' is as specified in section 8.2.7.1.1"
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
+
+    MATCH_outputEPCClass:
+      name: MATCH_outputEPCClass
+      description: "If this parameter is specified, the result will only include events that (a) have an `outputQuantityList` field (that is, `TransformationEvent` or extension event types that extend it); and where (b) one of the EPC classes listed in the `outputQuantityList` field (depending on event type) matches one of the EPC patterns or URIs specified in this parameter. The meaning of 'matches' is as specified in section 8.2.7.1.1"
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
```

Comment:
```suggestion
          format: uri
```

Reply:
Thanks @jmcanterafonseca-iota  - I've now corrected those upper-case URI to lower-case uri